CLAC

```diff
@@ -0,0 +1,157 @@
+                       DEDIS / EPFL / Lausanne / CH
+  Software Grant and Corporate Contributor License Agreement ("Agreement")
+                      http://www.apache.org/licenses/
+
+   Thank you for your interest in software from DEDIS (Lab).
+   In order to clarify the intellectual property license
+   granted with Contributions from any person or entity, the Lab
+   must have a Contributor License Agreement (CLA) on file that has been
+   signed by each Contributor, indicating agreement to the license terms
+   below. This license is for your protection as a Contributor as well
+   as the protection of the Lab and its users; it does not change
+   your rights to use your own Contributions for any other purpose.
+
+   This version of the Agreement allows an entity (the "Corporation") to
+   submit Contributions to the Lab, to authorize Contributions 
+   submitted by its designated employees to the Lab, and to grant 
+   copyright and patent licenses thereto.
```

Comment:
No, this is in the case an employee from another corporation wants to contribute. Then his corporation has to sign off that this employee is allowed to contribute. When the corporation signs off on the `CLAC`, it will grant its copyright and patent licenses to the `Lab`, which is DEDIS.

If you, Nicolas, want to contribute in your work-time to a project that needs a `CLA`, then EPFL needs to sign off, too, because you're under a working contract and cannot give your copyright (which belongs to EPFL) to somebody else.